docs_src/microservices/application/AdvancedTopics.md

```diff
@@ -369,7 +369,7 @@ When running in insecure mode, the secrets are stored and retrieved from the *Wr
 
 #### Getting Secrets
 
-Application Services can retrieve their secrets from their SecretStore      using the  [interfaces.ApplicationService.GetSecret()](#TBD) API or from the [interfaces.AppFunctionContext.GetSecret()](ContextAPI.md#getsecret) API  
+Application Services can retrieve their secrets from their SecretStore      using the  [interfaces.ApplicationService.GetSecret()](#TBD) API or from the [interfaces.AppFunctionContext.GetSecret()](AppFunctionContextAPI.md#getsecret) API  
```

Comment:
what is #TBD? It should point to some section in this document or something else?

Reply:
Good catch. Section didn't exist till now. Will update.